recipes/poco/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "poco" %}
+{% set version = "1.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pocoproject/poco/archive/poco-{{ version }}-release.tar.gz
+  sha256: 45ec5c759d2ad02bb812745f6cb585aa04af025c921b1bb4bdefa108ce4f1756
+
+build:
+  number: 0
+  skip: true    # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.0
+  host:
+    - openssl
+  run:
+    - openssl
```

Comment:
No need for openssl in `run` https://github.com/conda-forge/openssl-feedstock/blob/master/recipe/meta.yaml